guides/micronaut-aws-secretsmanager/micronaut-aws-secretsmanager.adoc

```diff
@@ -0,0 +1,107 @@
+= @guideTitle@
+
+@guideIntro@
+
+Authors: @authors@
+
+Micronaut Version: @micronaut@
+
+include::{commondir}/common-gettingStarted.adoc[]
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-create-app.adoc[]
+
+== OAuth 2.0 Dependency
+
+To use OAuth 2.0 integration, add the next dependency:
```

Comment:
s/ `add the next dependency` / `add the following dependency` /

Reply:
https://github.com/micronaut-projects/micronaut-guides/pull/290